src/cli/internal/utils/version_checker.go

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) CloudCover
+// SPDX-License-Identifier: Apache-2.0
+
+package utils
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/charmbracelet/log"
+	"github.com/cldcvr/terrarium/src/cli/internal/config"
+	"github.com/cldcvr/terrarium/src/pkg/db"
+	"github.com/rotisserie/eris"
+	"github.com/spf13/cobra"
+)
+
+var re = regexp.MustCompile(`github\.com/([^/]+)/([^/]+)`)
+
+type UpdateInfo struct {
+	UpdateRequired       bool
+	CurrentVersion       string
+	LatestReleaseVersion string
+}
+
+func IsFarmUpdateRequired(cmd *cobra.Command, args []string) {
+	updateInfo, _ := isUpdateRequired(config.FarmDefault())
+	if updateInfo != nil && updateInfo.UpdateRequired {
+		log.Info("Farm update available", "CurrentVersion", updateInfo.CurrentVersion, "LatestVersion", updateInfo.LatestReleaseVersion)
```

Comment:
Since, in the other json responses we consistently use camelCase, it'd be nice to follow the same convenience here by changing the PascalCase to camelCase.

```suggestion
		log.Info("Farm update available", "currentVersion", updateInfo.CurrentVersion, "latestVersion", updateInfo.LatestReleaseVersion)
```